packages/foundry-sdk-generator/src/generate/betaClient/generatePackage.ts

```diff
@@ -82,12 +72,7 @@ export async function generatePackage(
   };
 
   if (!options.beta) {
-    await generateClientSdkVersionOneDotOne(
-      ontology,
-      `typescript-sdk/${options.packageVersion} ${USER_AGENT}`,
-      hostFs,
-      packagePath,
-    );
+    throw new Error("Only beta is supported in this version");
```

Comment:
Once we have the ability to just get our beta releases into dev console, we can just remove this flag right?

Reply:
Correct, this is just hold over